bin/servicerouter.py

```diff
@@ -27,85 +29,87 @@
 Usage:
   $ servicerouter.py --marathon http://marathon1:8080 --haproxy-config /etc/haproxy/haproxy.cfg
 
-  The user servicerouter is executed as must have the right to reload haproxy.
+  The user that executes servicerouter must have the permission to reload
+  haproxy.
 
 
 Environment Variables:
   HAPROXY_GROUP
-  Only servicerouter instances which are members of the given group will
-  point to the service. Service routers with the group '*' will collect
-      all groups.
+    The group of servicerouter instances that point to the service.
+    Service routers with the group '*' will collect all groups.
 
   HAPROXY_{n}_VHOST
-  HTTP Virtual Host proxy hostname to catch
-  Ex: HAPROXY_0_VHOST = 'marathon.mesosphere.com'
+    The Marathon HTTP Virtual Host proxy hostname to gather.
+    Ex: HAPROXY_0_VHOST = 'marathon.mesosphere.com'
 
   HAPROXY_{n}_STICKY
-  Use sticky request routing for the service
-  Ex: HAPROXY_0_STICKY = true
+    Enable sticky request routing for the service.
+    Ex: HAPROXY_0_STICKY = true
 
   HAPROXY_{n}_REDIRECT_TO_HTTPS
-  Redirect HTTP traffic to https
-  Ex: HAPROXY_0_REDIRECT_TO_HTTPS = true
+    Redirect HTTP traffic to HTTPS.
+    Ex: HAPROXY_0_REDIRECT_TO_HTTPS = true
 
   HAPROXY_{n}_SSL_CERT
-  Use the given SSL cert for TLS/SSL traffic
-  Ex: HAPROXY_0_SSL_CERT = '/etc/ssl/certs/marathon.mesosphere.com'
+    Enable the given SSL certificate for TLS/SSL traffic.
+    Ex: HAPROXY_0_SSL_CERT = '/etc/ssl/certs/marathon.mesosphere.com'
 
   HAPROXY_{n}_BIND_ADDR
-  Bind to the specific address for the service
-  Ex: HAPROXY_0_BIND_ADDR = '10.0.0.42'
+    Bind to the specific address for the service.
+    Ex: HAPROXY_0_BIND_ADDR = '10.0.0.42'
 
   HAPROXY_{n}_MODE
-  Set the connection mode to either tcp or http. Default is tcp.
-  Ex: HAPROXY_0_MODE = 'http'
+    Set the connection mode to either TCP or HTTP. The default is TCP.
+    Ex: HAPROXY_0_MODE = 'http'
```

Comment:
Not a text comment. Just because we had an issue for that in Marathon: Does it also allow HTTPS? I know that you could argue that this is unnecessary but people have requested it for Marathon.